hibernate-core/src/main/java/org/hibernate/boot/model/internal/PropertyBinder.java

```diff
@@ -292,8 +292,9 @@ private void callAttributeBinders(Property property, Map<String, PersistentClass
 								: persistentClasses.get( holder.getEntityName() );
 				binder.bind( containingAnnotation, buildingContext, persistentClass, property );
 			}
-			catch ( Exception e ) {
-				throw new AnnotationException( "error processing @AttributeBinderType annotation '" + containingAnnotation + "'", e );
+			catch (Exception e) {
+				throw new AnnotationException( "error processing @AttributeBinderType annotation '" + containingAnnotation + "' for property '"
```

Comment:
```suggestion
				throw new AnnotationException( "error processing @AttributeBinderType annotation '" + containingAnnotation + "' for attribute '"
```

Reply:
I can also squash the commits if preferred.